metaflow/plugins/argo/test/live_run_failure_flow.py

```diff
@@ -0,0 +1,46 @@
+from metaflow import FlowSpec, step
+import time
+
+
+class LiveRunFailureFlow(FlowSpec):
+    """
+    This flow fails. Use this flow to test trigger_live_run function's handling
+    of failure.
+    """
+
+    @step
+    def start(self):
+        """
+        This step waits 5 seconds to let the trigger function test status.
+        """
+        print("LiveRunFailureFlow is starting")
+        time.sleep(5)
+        self.next(self.middle_step)
+
+    @step
+    def middle_step(self):
+        """
+        This step is a middle step and prints one line.
+        """
+        print("LiveRunFailureFlow is in its middle_step")
+        self.next(self.failure_step)
```

Comment:
`middle_step` can be removed. Apart from shorter code it saves a slight amount of time and cost per test run.